index.html

```diff
@@ -0,0 +1,1305 @@
+                  Biden's Missouri Lead Over Sanders Fueled by Over-50s and Non-Whites: Poll
+                </h5>
+                <p class="card-text mb-3">The former vice president is ahead of the Vermont senator in Missouri, a poll
+                  shows, but his lead is within the margin of error.</p>
+              </div>
+            </div>
+
+            <h5 class="text-muted mt-3 col-sm-12">CULTURE & TRAVEL</h5>
+            <div class="card-deck">
+              <div class="card travel w-100 ">
+                <img src="images/deck_img.jpg" alt="CUL_Uncharted_Stargazing_Banner"
+                  class="mapping-mobile lazy lazyloaded img-fluid w-100">
+                <div class="card-body pl-0">
+                  <h4 class="card-title culture-text pl-0">Best Stargazing Spots Around the Globe</h4>
+                  <p class="card-text pl-0">Astrotourism is taking off, and these are the best places—on our planet, at
+                    least—to witness the dazzling display of luminous balls of gas that we call stars</p>
+                </div>
+              </div>
+              <div class="card w-100  travel">
+                <img src="images/deck_img.jpg" alt="CUL_Uncharted_Stargazing_Banner"
+                  class="mapping-mobile lazy lazyloaded img-fluid w-100">
+                <div class="card-body pl-0">
+                  <h4 class="card-title culture-text pl-0">Best Stargazing Spots Around the Globe</h4>
+                  <p class="card-text pl-0">Astrotourism is taking off, and these are the best places—on our planet, at
+                    least—to witness the dazzling display of luminous balls of gas that we call stars</p>
+                </div>
+              </div>
+              <div class="card w-100 travel sticky-top">
+                <img src="images/deck_img.jpg" alt="CUL_Uncharted_Stargazing_Banner"
+                  class="mapping-mobile lazy lazyloaded img-fluid w-100">
+                <div class="card-body pl-0">
+                  <h4 class="card-title culture-text p-0">Best Stargazing Spots Around the Globe</h4>
+                  <p class="card-text p-0">Have plans to travel to Southeast Asia soon? Here's what our correspondent
+                    experienced during his monthlong stint in the region in February.</p>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <h5 class="text-muted more-stories pb-3">MORE STORIES</h5>
+          <div class="media mt-3 more-stories">
+            <img src="images/deck_img.jpg" class="mapping-small lazysize lazyloaded mr-3" alt="newsweek-img">
+
+            <div class="media-body">
+              <h5 class="media-title">More Aggressive' Coronavirus Strain May Have Evolved—But Don't Worry</h5>
+              <p class="d-md-block">
+                Scientists studied the viral DNA from 103 people infected with the virus that has killed over 3,000
+                people worldwide.
```

Comment:
IN the small breakpoint, the text in this section is too large. Please fix it.

Reply:
fixed